crates/polars-core/src/series/implementations/datetime.rs

```diff
@@ -356,20 +356,32 @@ impl SeriesTrait for SeriesWrap<DatetimeChunked> {
             .max_as_series()
             .into_datetime(self.0.time_unit(), self.0.time_zone().clone()))
     }
+
     fn min_as_series(&self) -> PolarsResult<Series> {
         Ok(self
             .0
             .min_as_series()
             .into_datetime(self.0.time_unit(), self.0.time_zone().clone()))
     }
+
+    fn median_as_series(&self) -> PolarsResult<Series> {
+        Series::new(self.name(), &[self.median().map(|v| v as i64)]).cast(self.dtype())
+    }
+
+    fn var_as_series(&self, _ddof: u8) -> PolarsResult<Series> {
```

Comment:
This should be removed because we want to raise if not supported for a dtype.

Reply:
I can add that to this PR, although I was going to implement this in my next PR anyway.